tracing-subscriber/src/fmt/mod.rs

```diff
@@ -304,14 +304,14 @@ where
     /// Returns an Error if the initialization was unsuccessful, likely
     /// because a global subscriber was already installed by another
     /// call to `try_init`.
-    pub fn try_init(self) -> Result<(), impl Error + Send + Sync + 'static> {
-        #[cfg(feature = "tracing-log/std")]
+    pub fn try_init(self) -> Result<(), Box<dyn Error + Send + Sync + 'static>> {
+        #[cfg(feature = "tracing-log")]
         tracing_log::LogTracer::init().map_err(Box::new)?;
```

Comment:
My guess is that we can probably remove the `Box::new` if we change the return type to `Box<dyn Error + ...>`, there should be an `Into` conversion that happens magically...

Reply:
Yep, you're right about that. Removed.